src/server/api/workLogs.ts

```diff
@@ -136,7 +136,7 @@ router.delete('/', (req, res) => {
       success: true,
       deletedWorkLog: result,
     });
-    logger.info('res.json success in workLogs.ts delete');
+    // logger.info('res.json success in workLogs.ts delete');
```

Comment:
Instead of commenting these out, you could also change them to `logger.debug` to lower the "log level" so that they don't show up in the logs unless you want them to.